src/modules/75-cf/strings/strings.en.yaml

```
@@ -507,6 +507,9 @@ pipeline:
     toTargetGroup: to target group
     toTargetGroups: to target groups
     pleaseSelectAFeatureFlag: Please select an environment and feature flag.
+    noFlagChanges: Please add a flag change.
+    removeFlagChange: Remove flag change
+    addFlagChange: Add Flag Change
```

Should this match removeFlagChange?
```suggestion
    addFlagChange: Add flag change
```

---

This capitalisation matches the mocks, so that's why I went that way. Honestly, I don't flip'n know what's right these days lol

---

sure I know we don't have consistency across the module but the least we can do is be consistent within a section/a page?

---

I'll change it coz I agree with you, but I suppose the difference is that the other uses are in a bit of info text and the other in an unseen aria attribute and then this one is used in a call to action button, so their intent is different.